clients/src/main/java/org/apache/kafka/common/utils/ThreadUtils.java

```diff
@@ -52,4 +55,26 @@ public Thread newThread(Runnable r) {
             }
         };
     }
+
+    /**
+     * Shuts down an executor service with a timeout. After the timeout/on interrupt, the service is forcefully closed.
+     * @param executorService The service to shut down.
+     * @param timeout The timeout of the shutdown.
+     * @param timeUnit The time unit of the shutdown timeout.
+     * @return True, if the shutdown was clean.
+     */
+    public static boolean shutdownExecutorServiceSilently(ExecutorService executorService,
```

Comment:
Since the return value is never used and similar methods (like org.apache.kafka.common.utils.Utils#closeQuietly) are void, we can make this void too.

nit: please rename it to shutdownExecutorServiceQuietly to keep a similar naming to others.

Reply:
thanks for the review, I fixed the method